src/components/CCPDashboard/PatientOverview.tsx

```diff
@@ -13,23 +12,19 @@ import {
   TableRow,
   TableCell,
 } from '@material-ui/core';
-import { useQuery } from '@apollo/react-hooks';
 import { Colours } from '../../styles/Constants';
-import {
-  FETCH_ALL_PATIENTS,
-  Patient,
-  TriageLevel,
-  Status,
-} from '../../graphql/queries/patients';
-import { TriageTag } from './TriageTag';
-import { PatientInfoTable } from './PatientInfoTable';
+import { Patient, Status } from '../../graphql/queries/patients';
+import { TotalPatientCard } from './TotalPatientCard';
+import { TriageCard } from './TriageCard';
+import { PatientInfoTableWithFilters } from './PatientInfoTableWithFilters';
 import { ScanPatientButton } from './ScanPatientButton';
-import LoadingState from '../common/LoadingState';
 
 interface PatientOverviewProps {
   // eslint-disable-next-line react/no-unused-prop-types
   eventId: string;
+  // eslint-disable-next-line react/no-unused-prop-types
   ccpId: string;
```

Comment:
just double checking we do need these for sure?

Reply:
They're required because the **edit patient profile page** has the route `/events/[event_id]/ccps/[ccp_id]/new` according to [this spreadsheet](https://docs.google.com/spreadsheets/d/1JTQlflqYCW8p78ceiyYTEJxTe1UuTgLT7OhHtPeyyV8/edit#gid=1720400799). `eventId` and `ccpId` are needed to be passed to `PatientInfoTable` > `PatientDetailsDialog` for the "Edit" button in that component which should link to the **edit patient profile page**.